test/test_adrv9002_p.py

```diff
@@ -139,8 +139,10 @@ def test_adrv9002_interface_gain_narrowband(
     from adi.adrv9002 import adrv9002
 
     sdr = adrv9002(iio_uri)
-    if sdr.rx0_sample_rate > 1000000:
-        pytest.skip("Baseband RX Sample Rate should be less than 1MHz to run this test.")
+    if (sdr.rx0_sample_rate or sdr.rx1_sample_rate) > 1000000:
+        pytest.skip(
+            "Baseband RX Sample Rate should be less than 1MHz to run this test."
+        )
```

Comment:
This is still not quite there... What I meant (even though unlikely) is that 1 channel can pass the test while the other might skip it... So, I think it should look something like:

```
if attr == "interface_gain_chan0":
    if sdr.rx0_sample_rate > 1000000:
        pytest.skip("some message")
else:
    if sdr.rx1_sample_rate > 1000000:
        pytest.skip("some message")
```

See the difference?

Reply:
Hello, Nuno. I now get your point. I initially assumed that it would just automatically read the property from the channel it tests, not all of the device properties, but apparently, it's not the case. That's why I made the initial conditional to be very simple. I didn't know that I actually made a loophole for errors in the test. I'm sorry. I was mistaken. It does read all of the device properties at each test run, albeit it's testing an attribute in channel 1 or channel 2. Thank you.

Reply:
As for the commits, I also rebased them. We won't have any problems with a polluted history.